frontend/components/page-contents/user-settings/LeftUserSettings.content.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="wrapper">
+    <BaseSpinner v-if="$fetchState.pending" />
+
+    <div class="left-content" v-if="user">
+      <div class="left-content-item edition-user-info-component">
+        <EditionUserInfoComponent :userInfo="user" />
+      </div>
+      <div class="left-content-item user-token-component">
+        <UserTokenComponent :userToken="user.api_key" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { ObservationDataset } from "@/models/Dataset";
+
+export default {
+  name: "LeftUserSettingsContent",
+  data() {
+    return {
+      user: null,
+    };
+  },
+  async fetch() {
+    await this.fetchUserInfo();
+  },
+  methods: {
```

Comment:
You have already this information in Vuex, you can get all user info with this.$auth.user